schematic/utils/google_api_utils.py

```diff
@@ -145,3 +154,25 @@ def export_manifest_csv(file_path: str, manifest: Union[pd.DataFrame, str]) -> N
         manifest.to_csv(file_path, index=False)
     else:
         export_manifest_drive_service(manifest, file_path, mime_type="text/csv")
+
+
+@retry(
+    stop=stop_after_attempt(5),
+    wait=wait_chain(
+        *[wait_fixed(1) for i in range(2)]
+        + [wait_fixed(2) for i in range(2)]
+        + [wait_fixed(5)]
+    ),
+    retry=retry_if_exception_type(HttpError),
+    reraise=True,
```

Comment:
See this doc for reraise:
https://tenacity.readthedocs.io/en/latest/#error-handling


```
Error Handling

Normally when your function fails its final time (and will not be retried again based on your settings), a RetryError is raised. The exception your code encountered will be shown somewhere in the middle of the stack trace.

If you would rather see the exception your code encountered at the end of the stack trace (where it is most visible), you can set reraise=True.
```